src/main/resources/application.yml

```
@@ -23,7 +23,13 @@ spring:
       name: ${SECURITY_USER_NAME}
       password: ${SECURITY_USER_PASSWORD}
       roles: ${SECURITY_USER_ROLES}
+  # SendGrid API Key configuration. Em produção, defina SENDGRID_API_KEY como variável de ambiente ou utilize um gerenciador de segredos.
+  sendgrid:
+      api-key: FAKE_TEST_KEY
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)